hooks/checkout

```diff
@@ -336,7 +336,7 @@ checkout() {
     # been force pushed in the meantime. Exit with ESTALE to signify the stale
     # branch reference in that case.
     if [[ "${exit_code}" -eq 128 ]]; then
-      log_info "Commit SHA ${BUILDKITE_COMMIT} does not exists. The ref might have been force pushed."
+      log_info "Fail to fetch commit:${BUILDKITE_COMMIT}. Check githublog under Artifacts tab for details."
```

Comment:
should it be `tmp/githublog.xxxxxxxxxx.log`?

Reply:
Even better, can use the actual name from `git_log`.